drivers/native-driver/src/mingwX86Main/kotlin/com/squareup/sqldelight/drivers/native/util/PoolLock.kt

```diff
@@ -0,0 +1,81 @@
+package com.squareup.sqldelight.drivers.native.util
```

Comment:
Does this file differ from the one that was moved? Why does it need two copies whereas the `NativeCache.kt` and `Stately.kt` files can stay in `mingwMain/`?

Reply:
No, this is the same file. Without this change project will not compile because of missing `platform.posix.*` imports.

I don't like this solution as well, but this is the only way I overlooked here: 
https://github.com/touchlab/SQLiter/blob/main/sqliter-driver/src/mingwX64Main/kotlin/co/touchlab/sqliter/concurrency/Lock.kt

Reply:
Ah, I see. So it's a commonizer problem. We'll revisit at some point in the future and hopefully it'll be fixed.